techniques/html/H101.html

```diff
@@ -97,29 +97,29 @@ <h2 class="box-h box-h-icon">About this Technique</h2>
+      <p>ランドマーク領域は、特定のセマンティック HTML 要素が使用されると暗黙的に作成される。これは、<a href="../aria/ARIA11">ARIA を用いて明示的にランドマークを作成する</a>場合とは異なる。これらの要素は次のとおりである:</p>
       <ul>
-        <li><code class="language-html">&lt;header&gt;</code>: When a <code class="language-html">header</code> is a direct child of the <code class="language-html">body</code> element, it creates a <code>banner</code> role, a region that typically contains the site's logo, name, and other persistent site-wide content at the top of a page.</li>
-        <li><code class="language-html">&lt;nav&gt;</code>: A region that contains navigation links to other pages or different parts of the same page.</li>
-        <li><code class="language-html">&lt;main&gt;</code>: A region that contains a page's main content.</li>
-        <li><code class="language-html">&lt;section&gt;</code>: A region that contains a generic section of a document or application. A <code class="language-html">section</code> element isn't exposed as a landmark region unless it has an accessible name.</li>
-        <li><code class="language-html">&lt;form&gt;</code>: A region of the document that represents a collection of form-associated elements, some of which can represent editable values that can be submitted to a server for processing. A <code class="language-html">form</code> element isn't exposed as a landmark region unless it has an accessible name.</li>
-        <li><code class="language-html">&lt;aside&gt;</code>: A region of the document that supports the main content, yet is separate and meaningful on its own.</li>
-        <li><code class="language-html">&lt;footer&gt;</code>: A region that, when its nearest ancestor is the <code class="language-html">body</code> element, contains information about the page such as copyrights and links to privacy statements.</li>
+        <li><code class="language-html">&lt;header&gt;</code>: <code class="language-html">header</code> が <code class="language-html">body</code> 要素の直接の子要素である場合、これは<code>banner</code> ロールを作成し、通常、サイトのロゴ、名前、その他のサイト全体の永続的なコンテンツを含む領域で、ページ上部に表示される。</li>
+        <li><code class="language-html">&lt;nav&gt;</code>: 他のページ又は同じページ内の異なる部分へのナビゲーションリンクを含む領域。</li>
+        <li><code class="language-html">&lt;main&gt;</code>: ページのメインコンテンツを含む領域。</li>
+        <li><code class="language-html">&lt;section&gt;</code>: ドキュメントまたはアプリケーションの一般的なセクションを含む領域。<code class="language-html">section</code> 要素は、アクセシブルな名前 (name) を持たない限り、ランドマーク領域として公開されない。</li>
+        <li><code class="language-html">&lt;form&gt;</code>: フォーム関連要素のコレクションを表すドキュメントの領域。これらの要素の一部は、サーバーに送信して処理できる編集可能な値を表すことができる。<code class="language-html">form</code> 要素は、アクセシブルな名前を持たない限り、ランドマーク領域として公開されない。</li>
+        <li><code class="language-html">&lt;aside&gt;</code>: メインコンテンツをサポートしながらも、それ自体で独立した意味を持つドキュメントの領域。</li>
+        <li><code class="language-html">&lt;footer&gt;</code>: 最も近い祖先が <code class="language-html">body</code> 要素である場合、著作権やプライバシーステートメントへのリンクなど、ページに関する情報を含む領域。</li>
```

Comment:
「著作権へのリンク」があるように読めてしまうので、少し訳を調整しました。
```suggestion
        <li><code class="language-html">&lt;footer&gt;</code>: 最も近い祖先が <code class="language-html">body</code> 要素である場合、著作権表示やプライバシーステートメントへのリンクなど、ページに関する情報を含む領域。</li>
```